components/sensor/emontx.rst

```diff
@@ -0,0 +1,308 @@
+OpenEnergyMonitor EmonTx Sensors
+================================
+
+.. seo:: :description: Instructions for setting up OpenEnergyMonitor EmonTx energy monitors with ESPHome. :image: emontx.jpg :keywords: EmonTx, OpenEnergyMonitor, energy monitor, power monitoring, CT clamp
+
+.. _emontx-component:
+
+Component/Hub
+-------------
+
+The ``emontx`` component allows you to use an EmonTx device with ESPHome.
+This component is a global hub that establishes the connection to the EmonTx via :ref:`UART <uart>` and translates the received data.
+Using the :ref:`emontx sensors <emontx-sensors>`, you can then create sensors for Home Assistant that track voltage, current, power, and temperature readings from the EmonTx.
+
+This component can also be used to send data to an `emoncms <https://emoncms.org/>`_ system via HTTP or MQTT. Working directly with emoncms seamlessly is a key benefit of this component, allowing you to integrate your energy monitoring data with powerful visualization and analysis tools.
+
+.. figure:: images/emontx5.jpg
+    :alt: OpenEnergyMonitor EmonTx5
+    :align: center
+    :width: 50.0%
+
+    OpenEnergyMonitor EmonTx5.
```

Comment:
just `emonTX` - remove the 5 as it will work with any emonTX as they all have a UART of some sort.